test/forkMainnet/SmartOrderStrategy/AMMs.t.sol

```diff
@@ -37,7 +37,8 @@ contract AMMsTest is SmartOrderStrategyTest {
         operations[0] = ISmartOrderStrategy.Operation({
             dest: UNISWAP_SWAP_ROUTER_02_ADDRESS,
             inputToken: defaultInputToken,
-            inputRatio: 0, // zero ratio indicate no replacement
+            numerator: 0, // zero ratio indicate no replacement
+            denominator: 10000,
```

Comment:
All `denominator` in zero ratio cases can be set to `0`:
```suggestion
            numerator: 0, // zero ratio indicate no replacement
            denominator: 0,
```

Reply:
ACK